packages/workflow/src/documents.ts

```diff
@@ -43,3 +48,107 @@ export async function uploadBase64ToMinio(
 
   return docUploadResponse.refUrl
 }
+
+export async function uploadBase64AttachmentsToDocumentsStore(
```

Comment:
I noticed this function in here as well: packages/gateway/src/features/registration/utils.ts. So when are we doing it in gateway & when in workflow?

Reply:
is it for the correction routes only for now?

Reply:
Yea exactly 👍  A while since I wrote this but I think the idea was that sooner or later we'd remove either or both if we could bake the logic into the `AttachmentInput` type at the GraphQL gateway level so all of it happens automatically. Would increase the amount of magic but minimise human error of forgetting to do this